code/modules/antagonists/cult/sword_fling.dm

```diff
@@ -16,7 +16,7 @@
 
 	cast_range = 6
 	active_msg = "You ready yourself to attempt to leap!"
-	var/obj/item/flinged_sword
+	var/obj/item/melee/cultblade/haunted/flinged_sword
```

Comment:
what happens to this ability in the case the sword gets deleted? u might end up with a hanging ref there

Reply:
it nulls the reference on destruction either way